oximeter/instruments/src/http.rs

```diff
@@ -129,15 +106,15 @@ impl LatencyTracker {
     /// to which the other arguments belong. (One is created if it does not exist.)
     pub fn update(
         &self,
-        request: &RequestInfo,
+        operation_id: &str,
         status_code: StatusCode,
         latency: Duration,
     ) -> Result<(), MetricsError> {
-        let key = RequestLatencyHistogram::key_for(request, status_code);
+        let key = operation_id.to_string();
```

Comment:
This isn't right. We still need this method, to have the `status_code` be part of the key. Without that, we'd combine latencies from calls to the same operation, regardless of whether they succeeded o failed.

Reply:
Fixed this in 0b647e035